CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 ### Removed
 
 - Remove support for enabling debugging mode with `OTEL_DOTNET_AUTO_DEBUG`.
+- Removed `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE` as a required environment
```

Comment:
@zacharycmontoya the StrongNamedTests (and a few others) rely on a "test instrumentation" that is introduced via a custom integrations file. I think there is a general desire to remove the integrations.json file, assuming that, it seems reasonable to either disable the affected tests or alternatively shipt a "test instrumentation" until we have another method to add extra instrumentations.

Reply:
Paolo: For now we have disabled the affected tests, and we can enable it again when we finish #2440

Rass: I'm not sure what you mean. Are there any changes you'd like to see?